src/Carbon/Traits/Difference.php

```diff
@@ -456,6 +464,14 @@ public function to($other = null, $syntax = null, $short = false, $parts = 1, $o
      *
      * Get the difference in a human readable format in the current locale from an other
      * instance given (or now if null given) to current instance.
+     *
+     * @param null $other
+     * @param null $syntax
+     * @param bool $short
+     * @param int  $parts
+     * @param null $options
```

Comment:
There are no null-types, it would mean you can only pass null as those parameters, please copy types and descriptions from `to`. Thanks. https://github.com/briannesbitt/Carbon/pull/1455/files#diff-39a84c4c7e44c741019124ce379a98b5R439